src/averageAges.js

```diff
@@ -37,7 +46,20 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const onlyWomen = woman => {
+    return (withChildren)
+      ? people.some(humem => humem.mother === woman.name)
```

Comment:
~humem~ human
please, install the Spell Checker [extension]( https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)